sdk/appconfiguration/app-configuration/src/index.ts

```diff
@@ -47,30 +47,51 @@ const deserializationContentTypes = {
 }
 
 export interface ETagOption {
+  /**
+   * Entity tag (etag) of the object
+   */
   etag?: string;
 }
 
-export type AddConfigurationSettingConfig = Pick<ModelConfigurationSetting, Exclude<keyof ModelConfigurationSetting, "key">>;
-export type AddConfigurationSettingOptions = ModelConfigurationClientCreateOrUpdateConfigurationSettingOptionalParams;
-export type AddConfigurationSettingsResponse = ModelCreateOrUpdateConfigurationSettingResponse;
+type CreateOrUpdateOptions = ModelConfigurationClientCreateOrUpdateConfigurationSettingOptionalParams;
+type CreateOrUpdateConfigurationSetting = Pick<ModelConfigurationSetting, Exclude<keyof ModelConfigurationSetting, "key">>;
+type CreateOrUpdateResponse = ModelCreateOrUpdateConfigurationSettingResponse;
+
+export type AddConfigurationSettingConfig = CreateOrUpdateConfigurationSetting;
+export type AddConfigurationSettingOptions = CreateOrUpdateOptions;
+export type AddConfigurationSettingsResponse = CreateOrUpdateResponse;
```

Comment:
I also just realized how silly this change is - it's a rename of what is _already_ being renamed above. Will fix.